### PR TITLE
Add is-all-rechable as related libraries

### PR DESCRIPTION
**Pull request by Kikobeats**
A subtle modification of this lib.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -64,7 +64,8 @@ Same as above except the `callback` doesn't have an `error` parameter.
 
 ## Related
 
-- [is-online](https://github.com/sindresorhus/is-online) - Check if the internet connection is up
+- [is-all-reachable](https://github.com/kikobeats/is-all-reachable) - Check if a list of hosts servers are all recheable.
```
Comment (silverwind) [inline readme.md]: Typo `recheable`

**Comment by sindresorhus** [comment]
What's the purpose and use-case of this? I don't get it. You can just call `is-reachable` module multiple times.

**Comment by Kikobeats** [comment]
@sindresorhus that's basically was the library do, but:

- in `is-reachable` you pass an array of hosts and it returns `true` if _any_ of the hosts are reachable.

- in `is-all-reachable` you pass an array of hosts and it returns `true` if _all_ of the hosts are reachable.

Feel free to merge or close this PR 😄

**Comment by silverwind** [comment]
Once promisifed, it'd be as simple as `Promise.all([isReachable(hostA), isReachable(hostB)])`.

**Comment by sindresorhus** [comment]
Closing as you can now simply do `Promise.all([hostA, hostB].map(isReachable))` to achieve what you want.

**Pull request closed without merging**